#
# Submitted-By: Erwin Rol, 2006-02-22
# Committed-By: Erwin Rol
#
# Error:
#
# i586-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I. -I../include -I../include -I../include -I../include -I../include -I../include -I../hw/xfree86/dixmods/extmod -I../hw/xfree86/os-support -isystem /home/erwin/projects/annax/devenv/visbox/local/i586-unknown-linux-gnu/include -isystem /home/erwin/projects/annax/devenv/visbox/local/i586-unknown-linux-gnu/usr/include -DHAVE_DIX_CONFIG_H -DXFree86Server -DXFree86LOADER -Wall -Wpointer-arith -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wnested-externs -fno-strict-aliasing -D_BSD_SOURCE -DHAS_FCHOWN -DHAS_STICKY_DIR_BIT -I/home/erwin/projects/annax/devenv/visbox/local/i586-unknown-linux-gnu/usr/include -I/home/erwin/projects/annax/devenv/visbox/local/i586-unknown-linux-gnu/usr/include/freetype2 -I../include -I../include -I../Xext -I../damageext -I../xfixes -I../Xi -I../mi -I../miext/shadow -I../miext/damage -I../render -I../randr -I../fb -g -O2 -MT fontcache.lo -MD -MP -MF .deps/fontcache.Tpo -c fontcache.c  -fPIC -DPIC -o .libs/fontcache.o
# fontcache.c:49:24: fontcacheP.h: No such file or directory
# fontcache.c:50:25: fontcachstr.h: No such file or directory
# fontcache.c:79: warning: no previous prototype for 'FontCacheExtensionInit'
#
#
# Description:
#
# The missing header files are in .../include/X11/extensions/ but the source files
# includes them as "fontcacheP.h" so they are not found. I am not sure if this
# is not an other configuration error, but this patch seems to work for now.
#
# State:
#
# Unknown.
#

--- xorg-server-X11R7.0-1.0.1/Xext/fontcache.c.orig	2006-02-22 01:10:48.000000000 +0100
+++ xorg-server-X11R7.0-1.0.1/Xext/fontcache.c	2006-02-22 01:11:20.000000000 +0100
@@ -46,8 +46,8 @@
 #include "inputstr.h"
 #include "servermd.h"
 #define _FONTCACHE_SERVER_
-#include "fontcacheP.h"
-#include "fontcachstr.h"
+#include <X11/extensions/fontcacheP.h>
+#include <X11/extensions/fontcachstr.h>
 #include <X11/Xfuncproto.h>
 
 #include "swaprep.h"
